src/main/java/org/broadinstitute/hellbender/tools/exome/CreatePanelOfNormals.java

```diff
@@ -39,13 +40,38 @@
  * </p>
  *
  * @author Valentin Ruano-Rubio &lt;valentin@broadinstitute.org&gt;
+ *
+ * <h3>Examples</h3>
+ * <p>
+ *     The command encompasses empirically determined parameters for TCGA project data.
```

Comment:
This is not accurate.  This has been run on many sources of data.  And in fact, much was done on internal test samples unrelated to TCGA.